products/llm_analytics/frontend/LLMAnalyticsTraceScene.tsx

```diff
@@ -69,6 +69,25 @@ enum TraceViewMode {
     Evals = 'evals',
 }
 
+// AI billing markup: 20% markup on top of cost
+const AI_COST_MARKUP_PERCENT = 0.2
```

Comment:
Question: Can we get it from backend somehow? So, if it changed there - it won't break this logic?